src/sagemaker/fw_utils.py

```diff
@@ -31,6 +31,9 @@
                                   'If you would like to use version {latest}, ' \
                                   'please add framework_version={latest} to your constructor.'
 
+EMPTY_FRAMEWORK_VERSION_ERROR = 'framework_version is required for this estimator. ' \
```

Comment:
it's not exactly accurate to say "this estimator" here since you can leave `framework_version` empty for `TensorFlow`...just provided that it's neither Python 3 nor script mode